tools/incubator.js

```diff
@@ -122,6 +125,9 @@ const Incubator = new Lang.Class({
         let file = this._download_file(asset['uri']);
         let info = file.query_info('*', Gio.FileQueryInfoFlags.NONE, null);
 
+        if (!info.get_content_type().startsWith('video'))
```

Comment:
This checks that the URI was for an actual video and not for example a youtube or dropbox page, right? If so +1 good to push.

Reply:
Yes, the idea is to double check that what we just downloaded is an actual video, or stop the script with a proper error (to avoid confusion).